pkg/api/copy_test.go

```diff
@@ -30,8 +30,8 @@ func TestDeepCopyApiObjects(t *testing.T) {
 	for i := 0; i < *fuzzIters; i++ {
 		for _, version := range []string{"", testapi.Version()} {
 			f := apitesting.FuzzerFor(t, version, rand.NewSource(rand.Int63()))
-			for kind := range api.Scheme.KnownTypes(version) {
-				item, err := api.Scheme.New(version, kind)
+			for kind := range api.Scheme.KnownTypes(api.Group, version) {
```

Comment:
We need to define a common function getGroupVersions() and use that here and at other places.
We can replace RegisteredVersions in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/registered/registered.go#L29 to RegisteredGroupVersions and use that.
It will have infra/v1, compute/v1, admin/v1, etc